src/components/ChatLog.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const chatEntries = props.entries.map((chatEntry, index) => {
+        return (
+            <ChatEntry
+                id={chatEntry.id}
+                sender={chatEntry.sender}
+                body={chatEntry.body}
+                timeStamp={chatEntry.timeStamp}
+                liked={chatEntry.liked}
+                onUpdateChatEntry={props.onUpdateChatEntry}
+                // every list needs a key identifier (not a prop)
```

Comment:
This would be a good place to add some sort of evev/odd split to account for which message should go on which side. You could use an `odd` prop, something like this:
```
odd={i % 2}
```